vcd/resource_vcd_vapp.go

```diff
@@ -4,24 +4,32 @@ import (
 	"fmt"
 	"log"
 	"regexp"
+	"strings"
 
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/vmware/go-vcloud-director/v2/govcd"
 	"github.com/vmware/go-vcloud-director/v2/types/v56"
 )
 
+const implicitVmInVappDeprecation = "Creation of vApp and an implicit single VM in the same structure is deprecated"
+const vAppUnknownStatus = "-unknown-status-"
+
 func resourceVcdVApp() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceVcdVAppCreate,
 		Update: resourceVcdVAppUpdate,
 		Read:   resourceVcdVAppRead,
 		Delete: resourceVcdVAppDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceVcdVappImport,
+		},
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "A unique name for the vApp",
```

Comment:
We should write unique at which scope, like Org VDC or whole Org?

Reply:
It's unique among the vApps.

Reply:
The question is whether the name needs to be unique across the whole Org or only inside VDC. This is useful information to figure out & put in Description and docs. Especially, given that our users query by names in our provider.

Reply:
Will change the description to "A name for the vApp, unique withing the VDC"

Reply:
done